docs/reference/extensions/spring-boot-framework.rst

```diff
@@ -0,0 +1,112 @@
+.. _spring-boot-framework-reference:
+
+spring-boot-framework
+---------------------
+
+The Spring Boot extension streamlines the process of building Spring Boot
+application rocks.
+
+The extension packs and copies the Jar package file to the rock.
+By default, the base ``bare`` is used, to generate a lightweight image.
+
+
+.. note::
+    The Spring Boot extension is compatible with the ``bare`` and
+    ``ubuntu@24.04`` bases.
+
+
+Project requirements
+====================
+
+To use the ``spring-boot-framework`` extension, there must be either:
```

Comment:
> ``spring-boot-framework``

Except for when we're referring to its name in code, let's please use the plain English name of the extension. :)